cmake/projects/qhull/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+#ok
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    qhull
+    VERSION
+    7.2.0-p0
+    URL
+    "https://github.com/t0p4/qhull/archive/v7.2.0-p0.tar.gz"
```

Comment:
I've created https://github.com/hunter-packages/qhull and branch `hunter-7.2.0`. Please send patches there, see:
* https://docs.hunter.sh/en/latest/creating-new/patch.html